exp/controllers/azuremachinepoolmachine_controller_test.go

```diff
@@ -132,13 +135,17 @@ func getAReadyMachinePoolMachineCluster() (*clusterv1.Cluster, *infrav1.AzureClu
 	}
 
 	cluster := &clusterv1.Cluster{
+		TypeMeta: metav1.TypeMeta{
+			Kind: "Cluster",
```

Comment:
Does the kind for these other objects need to be specified? I tend to lean toward not including them if they have no bearing on the result of the test.

Reply:
I'm pretty sure I added those because some of the tests that share this helper were failing because `AzureCluster` didn't have a Kind and we check the Kind value now in the controllers. I added them for all the objects returned by that function for consistency.